Ansible role that sets up a mariadb server instance

Scriptkiddi opened the pull request:
Added defaults that are equal to the default values from my-huge.cnf
besides one value that "skip-networking" is by default true so that
every network connection will be rejected when the server is run with
default values

Scriptkiddi committed:
Ansible role that setsup a mariadb server instance

Added defaults that are equal to the default values from my-huge.cnf
besides one value that "skip-networking" is by default true so that
every network connection will be rejected when the server is run with
default values

SlothOfAnarchy commented on README.md:
```diff
@@ -0,0 +1,40 @@
+# Role Name
```
set role name

Scriptkiddi replied:
done

SlothOfAnarchy commented on meta/main.yml:
```diff
@@ -0,0 +1,21 @@
+galaxy_info:
+  author: Fritz Otlinghaus
+  description: Ansible role to configure and start an mariadb server
```
"a" instead of "an"

SlothOfAnarchy commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,163 @@
+
+# The following options will be passed to all MySQL clients
+[client]
+#password   = your_password
```
add password var, maybe even mark it mandatory

SlothOfAnarchy commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,163 @@
+thread_concurrency = {{- mariadb_mysqld_thread_concurrency -}}
+
+# Point the following paths to a dedicated disk
+#tmpdir     = /tmp/
```
add tempdir var

SlothOfAnarchy commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,163 @@
+# Example MariaDB config file for very large systems.
```
mark as ansible_managed

SlothOfAnarchy commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,163 @@
+# Uncomment the following if you are using InnoDB tables
+#innodb_data_home_dir = @localstatedir@
+#innodb_data_file_path = ibdata1:2000M;ibdata2:10M:autoextend
+#innodb_log_group_home_dir = @localstatedir@
```
add innodb support

SlothOfAnarchy commented on tasks/main.yml:
```diff
@@ -0,0 +1,18 @@
+---
+# tasks file for .
```
remove or correct comment

SlothOfAnarchy commented on README.md:
```diff
@@ -0,0 +1,40 @@
+
+## Role Variables
+
+Read the mariadb doc to get to know what vars can be set
```
link?

SlothOfAnarchy commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,163 @@
+#master-host     =   <hostname>
+#
+# The username the slave will use for authentication when connecting
+# to the master - required
```
you **could** add vars for replication

Scriptkiddi committed:
Removed minus from all vars in template

Removed all of them since mariadb uses a fucking new line to determine
when a statment ends

Scriptkiddi committed:
reworked readme

dasJ commented on templates/my.cnf.j2:
```diff
@@ -0,0 +1,164 @@
+# but will not function as a master if omitted
+server-id   = {{ mariadb_mysqld_server_id }}
+
+# Replication Slave (comment out master section to use this)
```
@Scriptkiddi Why the fuck is there a huge commented out area?
